specification/schemas/Return.json

```diff
@@ -50,6 +50,9 @@
             },
             "shop": {
               "$ref": "#/components/schemas/ShopRelationship"
+            },
+            "return_method": {
+              "$ref": "#/components/schemas/ReturnMethodRelationship"
             }
```

Comment:
We need to make it required!

Reply:
@yoan-myparcel then it will break all return tests until the API is implemented.

Reply:
@yoan-myparcel opened this https://github.com/MyParcelCOM/api-specification/pull/881 so it can be merged after the API gets merged.